packages/account-v2/src/containers/PersonalDetailsForm/PersonalDetails.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { Fragment } from 'react';
+import { Field, FieldProps } from 'formik';
+import { useActiveTradingAccount, useGetAccountStatus, useSettings } from '@deriv/api-v2';
+import { CountrySelector } from '../../components/CountrySelector';
+import { DatePicker } from '../../components/DatePicker';
+import { FormInputField } from '../../components/FormFields';
+import { isFieldDisabled } from '../../utils/personalDetailsUtils';
+
+export const PersonalDetails = () => {
```

Comment:
**NIT:** Just to be safe, can we have to check to ensure this component is always wrapped with formik? U can make use of `useFormikContext()` and check if the reference is defined. U can check other sub forms we have in the package